layouts/partials/search.html

```diff
@@ -0,0 +1,88 @@
+<link href="/pagefind/pagefind-ui.css" rel="stylesheet">
+<script src="/pagefind/pagefind-ui.js"></script>
+
+<style>
```

Comment:
Would be nice to have this defined outside of the template, in the assets/css dir.

Reply:
Hmm, I added it under `assets/theme-css` instead of `doc/assets/css` since the search feature is provided by the theme and not just the example doc site.  That worked fine locally with the serve-dev make target, but it's blowing up in the CI build: https://app.netlify.com/sites/scientific-python-hugo-theme/deploys/66a68856fbc041000831976e

For now I'm reverting to put this CSS back where it was (inline) in the search partial, which worked, and affords the benefit of having (almost) the entire implementation in a single file, for easy delete or modify.